src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_time_field/components/advanced_options/advanced_options.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import {
+  EuiForm,
+  EuiFormRow,
+  EuiFieldText,
+} from '@elastic/eui';
+
+export const AdvancedOptions = ({
+  showingAdvancedOptions,
+  indexPatternId,
+  onChangeIndexPatternId,
+}) => {
+  if (!showingAdvancedOptions) {
+    return false;
+  }
+
+  return (
+    <EuiForm>
+      <EuiFormRow
+        label="Custom index pattern ID"
+        helpText={
+          <span>
+            Kibana will provide a unique identifier for each index pattern.
+            If you do not want to use this unique ID, enter a custom one.
+          </span>
+        }
+      >
+        <EuiFieldText
+          name="indexPatternId"
+          data-test-subj="createIndexPatternIdInput"
+          value={indexPatternId}
+          onChange={onChangeIndexPatternId}
+          placeholder="Id"
```

Comment:
Can we change this to `custom-index-pattern-id` so it looks more like something someone would use?

Reply:
Which part? `data-test-subj` or `placeholder`?

Reply:
Referring to the placeholder value:

```html
placeholder="custom-index-pattern-id"
```